src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js

```diff
@@ -59,18 +63,46 @@ const ReferenceSolutionStatus = ({ description, solution: { userId, createdAt },
             <UsersNameContainer userId={userId} />
           </td>
         </tr>
+
+        {Boolean(submittedBy) && submittedBy !== userId && (
+          <tr>
+            <td className="text-center">
+              <Icon icon="user" />
+            </td>
+            <th className="text-nowrap">
+              <FormattedMessage id="app.referenceSolution.reevaluatedBy" defaultMessage="Re-evaluated by" />:
+            </th>
+            <td>
+              <UsersNameContainer userId={submittedBy} showEmail="icon" />
+            </td>
+          </tr>
+        )}
+
+        {Boolean(environment) && Boolean(environment.name) && (
```

Comment:
Why checking `environment.name` here?

Reply:
Copied from `SolutionStatus` component... Probably can be omitted.

Reply:
I think it was just defensive programming.